src/streamlink/plugins/btv.py

```diff
@@ -60,8 +61,10 @@ def _get_streams(self):
         if stream_url == "geo_blocked_stream":
             log.error("The content is not available in your region")
             return
-
-        return HLSStream.parse_variant_playlist(self.session, stream_url)
+        
+        hls_url = update_scheme("hls://https://", stream_url, force=False)
+        streams = HLSStream.parse_variant_playlist(self.session, hls_url)
+        return streams or {"live": HLSStream(self.session, hls_url)}
```

Comment:
As you can see in #5698, the site started returning HLS multivariant playlists instead of HLS media playlists, so a fix was committed accordingly. The URL test cases returned valid streams (`./script/test-plugin-urls.py btv`), and they still do that now, I just checked it again.

Does your PR mean that this is not the case 100% of the time, and they return both kinds? You're saying that Streamlink is throwing errors. That shouldn't happen when trying to parse a multivariant playlist on a media playlist. The result should be an empty stream list.